packages/components/src/List/Toolbar/ColumnChooserButton/README.md

```diff
@@ -0,0 +1,274 @@
+|---|---|---|
+| children | React.element, [React.element] | the content of the popover
+| columns | array |  these columns comes from the list, they will populate the popover
+| filterValue | string | the current value of the filter
+| id | string | Use as prefix for all the children components
+| nbLockedLeftItems | number | the number of locked items you want, beginning at the left
+| submit | function | callback trigger when the form is submit, passed to the column manager hook (see hook)
+| t | function | i18n translate
+
+### ColumnChooserHeader
+
+The header of the column chooser.
+Title by default is "Modify columns". The number of selected columns is displayed here.
+You can pass children to the header to customize it.
+By default, it consumes the [columnChooserContext](#columnchoosercontext).
+
+| Props | Type | Info
+|---|---|---|
+| className | string | class passed to the tooltip header
+| children | React.element, [React.element] | the header's content of the popover
+
+### ColumnChooserFooter
+
+Footer of the column chooser.
+The apply button is here and it submit the form.
+You can pass children and override the default render.
+By default, it consumes the [columnChooserContext](#columnchoosercontext).
+
+The component holds reference :
+* Submit : [SubmitButton](#SubmitButton)
+* SelectAll: [SelectAllCheckbox](#SelectAllCheckbox)
+
+| Props | Type | Info
+|---|---|---|
+| className | string | class passed to the tooltip header
+| children | React.element, [React.element] | the footer  content of the popover
+
+### ColumnChooserBody
+
+Body of the column chooser.
+It renders the different column row.
+You can pass a function as children, it will receive the columns as arguments.
+Else, if children is not a function it will render normally.
+It consumes the [columnChooserContext](#columnchoosercontext).
+
+The component holds reference :
+* Row : [ColumnChooserRowRenderer](#columnchooserrowrenderer)
+
+| Props | Type | Info
+|---|---|---|
+| children | function | the body's  content of the popover, called with the columns as arguments.
+
+### ColumnChooserRowRenderer
+
+The row renderer only render its children.
+It holds reference to other components: , .
+By default, it consumes the [columnChooserContext](#columnchoosercontext).
+
+The component holds reference :
+* Visibility: [RowCheckbox](#rowcheckbox)
+* Label: [RowLabel](#rowlabel)
+
+| Props | Type | Info
+|---|---|---|
+| className | string | class passed to div wrapper row
+| children | React.element, [React.element] | the row's  content of the popover
+
+### RowCheckbox
+
+Add a checkbox to switch the visibility of a column.
+If the column is locked, a locked icon replaces the checkbox.
+	dataFeature,
+	describedby,
+	description,
+	id,
+	label,
+	locked = false,
+	onClick,
+	checked = false,
+
+| Props | Type | Info
+|---|---|---|
+| dataFeature | string | data-feature attribute of the input
+| describedby | string | the id of the markup holding the description
```

Comment:
I don't think we should set that in the api. You have a description prop just after, and if you don't want to use it, let's spread the rest instead of selecting them like that or you will need to add tons of other attributes in the proptypes.

Reply:
Props are required so have to be in documentation. Or I remove the required property.
Don't care.